objects/obj_projectile/Step_0.gml

```diff
@@ -2,9 +2,9 @@ x += hspd;
 y += vspd;
 
 var padding = 64;
-if !(point_in_rectangle(x,y,-padding,-padding,room_width+padding,room_height+padding)) {
-	instance_destroy(); //destroy if outside room
-}
+//if !(point_in_rectangle(x,y,-padding,-padding,room_width+padding,room_height+padding)) {
```

Comment:
Can you add the comment //Turned off for proc gen
just like you did for the player create.